docs/features/techdocs/configuring-ci-cd.md

```diff
@@ -96,3 +102,66 @@ npx @techdocs/cli publish --publisher-type <awsS3|googleGcs> --storage-name <buc
 
 The updated TechDocs site built in this workflow is now ready to be served by
 the TechDocs plugin in your Backstage app.
+
+## Example: GitHub Actions CI and AWS S3
+
+Here is an example workflow using GitHub Actions CI and AWS S3 storage. You can
+use any CI and other cloud storage providers - Google Cloud Storage and Azure
+Blob Storage.
+
+Add a `.github/workflows/techdocs.yml` file in your
+[Software Template(s)](../software-templates/index.md) like this -
+
+```yaml
+name: Publish TechDocs Site
+
+on:
+  push:
+    branches: [main]
+    # You can even set it to run only when TechDocs related files are updated.
+    # paths:
+    #   - "docs/**"
+    #   - "mkdocs.yml"
+
+jobs:
+  publish-techdocs-site:
+    runs-on: ubuntu-latest
+
+    # The following secrets are required in your CI environment for publishing files to AWS S3.
+    # e.g. You can use GitHub Organization secrets to set them for all existing and new repositories.
+    env:
+      TECHDOCS_S3_BUCKET_NAME: ${{ secrets.TECHDOCS_S3_BUCKET_NAME }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_REGION: ${{ secrets.AWS_REGION }}
+      ENTITY_NAMESPACE: 'default'
+      ENTITY_KIND: 'Component'
+      # In a Software template, Scaffolder will replace {{cookiecutter.component_id | jsonify}}
+      # with the correct entity name. This is same as metadata.name in the entity's catalog-info.yaml
+      ENTITY_NAME: '{{ cookiecutter.component_id | jsonify }}'
```

Comment:
Presume this value is only for a scaffoldable YAML? should this instead be static like 'my-doc-entity' here in this ?

Reply:
Hmm, I guess I could do this
```yaml
      ENTITY_NAME: 'my-doc-entity'
      # In a Software template, Scaffolder will replace {{cookiecutter.component_id | jsonify}}
      # with the correct entity name. This is same as metadata.name in the entity's catalog-info.yaml
      # ENTITY_NAME: '{{ cookiecutter.component_id | jsonify }}'
```


One would really add this YAML to their Software Template and so the template variable is important - thus wanted to highlight it.